src/test/kotlin/interactor/plus/GetCitiesNameWithCheapestTennisPriceOnWeekendInteractorTest.kt

```diff
@@ -0,0 +1,77 @@
+package interactor.plus
+
+import data.BananaCheapestFakeData
+import data.EmptyFakeData
+import data.InvalidFakeData
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.function.Executable
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal class GetCitiesNameWithCheapestTennisPriceOnWeekendInteractorTest {
+    private lateinit var getCitiesNameWithCheapestTennisPrice: GetCitiesNameWithCheapestTennisPriceOnWeekendInteractor
+
+    @BeforeEach
+    fun setup() {
+        getCitiesNameWithCheapestTennisPrice =
+            GetCitiesNameWithCheapestTennisPriceOnWeekendInteractor(BananaCheapestFakeData)
+    }
+
+    @Test
+    fun `should return correct 2 cities when the data is valid`() {
+        // given limit the return by 2
+        val limit = 2
```

Comment:
There is no need variable to limit use limit as hardCode

Reply:
nop to be generic as possible!

Reply:
to be more readable